NEXT_CHANGELOG.md

```diff
@@ -130,6 +130,21 @@ By [@neominik](https://github.com/neominik) in https://github.com/apollographql/
 
 ## 🛠 Maintenance
 
+### Add more details when GraphQL request is invalid ([Issue #2301](https://github.com/apollographql/router/issues/2301))
+
+Add more context to the error we're throwing if your GraphQL request is invalid, here is an exemple if you pass `"variables": "null"` in your JSON payload.
+```json
+{
+  "message": "Invalid GraphQL request",
+  "extensions": {
+    "details": "failed to deserialize the request body into JSON: invalid type: string \"null\", expected a map at line 1 column 100",
+    "code": "INVALID_GRAPHQL_REQUEST"
+  }
+}
+```
```

Comment:
Just to be clear, this would be nested within an error element inside the `errors` on the response — not as a top-level property — right?  (e.g., `{errors: [{ message: "Invalid GraphQL request", "extensions": ... }]}`) 

If nested, should/could we update this example to indicate that?

Reply:
yes [019b4ac](https://github.com/apollographql/router/pull/2306/commits/019b4ac189ba47b91b908d8be14c5e700df1915c) fixes it